main.py

```diff
@@ -280,6 +280,53 @@ def campaign_group_page(campaign_group=None, pwg=None):
                            pwg=pwg,
                            user_info=user_info)
 
+@app.route('/missing_page/<string:campaign_group>')
+@app.route('/missing_page/<string:campaign_group>')
+def missing_page(campaign_group=None, pwg=None):
+    conn = sqlite3.connect('twiki.db')
+    cursor = conn.cursor()
+    sql_args = [campaign_group]
+    sql_query = '''SELECT 1,
+                          dataset,
+                          ifnull(extension, ""),
+                          total_events,
+                          campaign,
+                          resp_group,
+                          cross_section,
+                          fraction_negative_weight,
+                          target_num_events
+                   FROM twiki_samples
+                   WHERE campaign = ?'''
+
+    if pwg and pwg in all_pwgs:
+        sql_query += ' AND interested_pwgs LIKE ?'
+        sql_args.append('%%%s%%' % (pwg))
+
+    rows = cursor.execute(sql_query, sql_args)
+    rows = [(get_short_name(r[1]),  # 0 Short name
+             r[1],  # 1 Dataset
+             r[2],  # 2 MiniAOD request
+             r[3],  # 3 total events
+             r[4],  # 4 campaign
+             r[5],  # 5 respective group
+             r[6],  # 6 cross section
+             r[7],  # 7 frac neg wgts
+             r[8],  # 7 target num events
+             [x for x in r[5].split(',') if x],  # 17 Interested pwgs
+            ) for r in rows]
+    logging.error('the following', rows)
```

Comment:
This looks like a very strange log message. It also misses `%s` in the format (first argument).